jib-maven-plugin/README.md

```diff
@@ -210,11 +209,6 @@ Then, you can build your container image by running:
 mvn package
 ```
 
-### Additional Build Artifacts
-
-As part of an image build, Jib also writes out the _image digest_ to
```

Comment:
Isn't this the default behavior that didn't change? If so, it's fine to leave this, just adding "... by default" and "the names (paths) are configurable?" You thought it's just good to remove this?

Reply:
It seemed a bit redundant after adding it to the "Extended Usage" section, considering those list the defaults.

Reply:
I think it's meaningful to call out the additional artifacts here. The "Extended Usage" is like an API reference that people don't look into it until necessary.